### DISCUSSION TITLE
Field select with relationship

### DISCUSSION THREAD

**Discussion by Burotica** [TiagoSilvaPereira/vemto-issues#301] [Ideas & Suggestions]
It is a shame not to be able to make a relation between two select fields.

I try for example to create a CMS with the creation of Pages which have a Category and a Sub-Category (in relation to the chosen Category)

**Comment by TiagoSilvaPereira** [comment]
Hi @Burotica it is already in my plans to add this feature to Vemto. For now, you can do it by editing the generated code
